tests/utils/websocket/decode.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package websocket
```

Comment:
Well it did, and the decode function was also a bit different. But it might just work, I merged them and removed the `websocket` package. The CI seems to pass so let's just assume this is resolved :wink: